### PR TITLE
feat: cs-43-06 get coverage

### PR DESCRIPTION
**Pull request by ronaldo-mendoza-jalafund**
## Related Tickets & US
- https://app.clickup.com/t/868auyhmw

## Description

Define the endpoint of the coveragre vector 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

image

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

- - -
This change is [Reviewable](https://reviewable.io/reviews/SoftDevSix/ArgosAPI/5)

### CODE REVIEW

File: src/main/java/com/softdevsix/domain/enums/Status.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.domain.enums;
+
+public enum Status {
```
Comment (0santisimo0) [inline src/main/java/com/softdevsix/domain/enums/Status.java:3]: Empty enum Class, Use CoverageStatus enum class that has the 3 status types

**Comment by 0santisimo0** [comment]
image
Why are there 2 "dtos" directories, sort the project, remove non-useful classes

File: src/main/java/com/softdevsix/domain/files/enums/IssueType.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.domain.files.enums;
+
+public enum IssueType {
```
Comment (0santisimo0) [inline src/main/java/com/softdevsix/domain/files/enums/IssueType.java:3]: This enum already exists in another directory, and this enum isn't complete, erase this class

File: src/main/java/com/softdevsix/domain/files/enums/Severity.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.domain.files.enums;
+
+public enum Severity {
```
Comment (0santisimo0) [inline src/main/java/com/softdevsix/domain/files/enums/Severity.java:3]: This enum already exists in another directory, and this enum isn't complete, erase this class

File: src/main/java/com/softdevsix/domain/pullrequests/PullRequest.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class PullRequest {
```
Comment (0santisimo0) [inline src/main/java/com/softdevsix/domain/pullrequests/PullRequest.java:12]: Empty entity, pull request will no longer be used, please check the new workflow. Information will not be received from a pull request now it is received from a static loading project.

File: src/main/java/com/softdevsix/domain/pullrequests/PullRequestCoverage.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class PullRequestCoverage {
```
Comment (0santisimo0) [inline src/main/java/com/softdevsix/domain/pullrequests/PullRequestCoverage.java:12]: Erase pullRequests Directory, non-useful Classes, this entities already exist

**Review by 0santisimo0 — CHANGES_REQUESTED** [review]
image (19)
[Duplicated classes and code, order the project erase un-useful classes]

**Comment by 0santisimo0** [comment]
image
Why Project, Issue, Coverage, FileCoverage or Project and enums are inside package "files" ?

**Comment by GabyGV-GGV** [comment]
Why is this pull request being closed?
@zamoranoalicia  Because this was a base implementation with the previous entities, an update was being performed, and due to time issues, it was done in another branch with the same idea of ​​the business logic, but with the updated entities.
Owner: @ronaldo-mendoza-jalafund 
Reviews: @0santisimo0

**Pull request closed without merging**